### DISCUSSION TITLE
ValueMappings: Support template variables in mapping condition and display text

### DISCUSSION THREAD

**Discussion by baur** [grafana/grafana#38308] [Dashboards, Panels & Visualizations]
**What would you like to be added**:

Please allow using a variable on Value mappings 

![image](https://user-images.githubusercontent.com/2898361/122694979-ec6a1280-d260-11eb-97a1-7b28515b9471.png)

**Why is this needed**:

I'd like to highlight the current text of StatPanel value depending on the variable value

![image](https://user-images.githubusercontent.com/2898361/122695681-f9880100-d262-11eb-8802-6c6f427e783c.png)

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
